document/travis-deploy.sh

```diff
@@ -9,18 +9,10 @@ TARGET_BRANCH="gh-pages"
 
 function doCompile {
   # TODO(littledan): Integrate with document/deploy.sh
-  cd document/js-api
+  cd document
   make
-  cd ../web-api
-  make
-  cd ../../out
-  if [[ ! -e js-api ]]; then mkdir js-api; fi
-  mv ../document/js-api/_build/html/index.html js-api/index.html
-  git add js-api/index.html
-  if [[ ! -e web-api ]]; then mkdir web-api; fi
-  mv ../document/web-api/_build/html/index.html web-api/index.html
-  git add web-api/index.html
   cd ../
+  mv document/_build/* out/
```

Comment:
Change out to _build below? Alternatively, add out to the clean target in the Makefile. But I'd prefer producing just one dir and the _xxx convention for generated dirs.

Reply:
Done